static/js/hotkey.js

```diff
@@ -565,10 +572,16 @@ export function process_hotkey(e, hotkey) {
         case "vim_right":
         case "tab":
         case "shift_tab":
+        case "open_inbox":
         case "open_recent_topics":
             if (recent_topics_util.is_in_focus()) {
                 return recent_topics_ui.change_focused_element($(e.target), event_name);
             }
+        case "open_inbox":
```

Comment:
This line should be deleted; it does nothing and makes the switch statement here very confusing. 

What we're doing is having a single block of code be the handler for all of these keys, and that handle is to dispatch to the recent topics and inbox modules' own code if appropriate, and that's much more clear if we just have two if statements in the block here.